bin/chandra_repro

```diff
@@ -137,6 +137,63 @@ def error_out(params,msg,val):
     print("\n%s" % msg)
     raise ValueError("%s" % val)
 
+
+def filter_aspsolobi_from_aspsol(asol_list,mydir):
+    """As of DS10.8.3, there are now two flavors of aspect solution 
+    files, per OBI (CONTENT=ASPSOLOBI) and the original per-constant
+    aspect interval (cai) (CONTENT=ASPSOL).  Users should only ever
+    have one or the other. 
+    
+    Unfortunately, due to a quirk in the archive, users may have both.
+    
+    We know that there can be 1 and only 1 ASPSOLOBI file per obi (multi-obi
+    obsid's will need to have already been split, #repro5). 
+    
+    So if we find both ASPSOLOBI and ASPSOL files, we are going to pick
+    the ASPSOLOBI files.  This provides a better off-axis angle value
+    and tighter FOV.
+    """
+    from pycrates import read_file
+
+    content_values = {}
```

Comment:
You could make this a dictionary with a default, which makes the addition bit simpler below. So at the top

```python
from collections import defaultdict
```

then here

```python
content_values = defaultdict(list)
```

so that later on you can just say

```python
content_values[content].append(f)
```

without having to check if `content` already exists

Reply:
Thanks.  Done.